logUpdater.py

```diff
@@ -49,6 +50,8 @@ def __init__(self, dirIn=None, dryRun=False, remote=None, webDir="/data/sdt/buil
             + self.release
         )
         self.ssh_opt = "-o CheckHostIP=no -o ConnectTimeout=60 -o ConnectionAttempts=5 -o StrictHostKeyChecking=no -o BatchMode=yes -o PasswordAuthentication=no"
+        if "cern.ch" not in socket.getfqdn():
+            self.ssh_opt = "-J cmsbuild@lxplus.cern.ch " + self.ssh_opt
```

Comment:
@aandvalenzuela , instead of using hard-coded `cmsbuild` here, can get is from `self.remote` ?